platforms-serverless/netlify-cli/prisma/schema.prisma

```diff
@@ -6,7 +6,7 @@ datasource db {
 
 generator client {
   provider      = "prisma-client-js"
-  binaryTargets = ["native", "rhel-openssl-1.0.x"]
+  binaryTargets = ["rhel-openssl-1.0.x"]
   output        = "../functions/generated/client"
```

Comment:
I think we should test closer to what users are doing by default, meaning we could remove 
`output        = "../functions/generated/client"`

Reply:
This is in the `netlify-cli` project, where a complete function directory is manually uploaded with the CLI `deploy`. I don't think without specifying an output directory will work there.